infrastructure/src/main/java/org/corfudb/infrastructure/LayoutServer.java

```diff
@@ -66,6 +79,7 @@
 
     /**
      * HandlerMethod for this server.
+     * [RM] Remove this after Protobuf for RPC Completion
      */
     @Getter
     private final HandlerMethods handler =
```

Comment:
Why can't you remove this in this patch? It's not needed. I think we should eagerly remove dead code paths because its hard to track them and try to delete them later.

Reply:
Note: It was made abstract in the other PR, so a getHandler method is required.